app/genre.rb

```diff
@@ -0,0 +1,14 @@
+class Genre
+  attr_reader :name, :items
```

Comment:
It should be better if you add atrr_reader for `id` now, you can not access the property to save it as JSON later.

Reply:
Hi @Nuri1977, I was following the UML class diagram, and it says it should be private and not accessible. Maybe I'm not reading it correctly, can you kindly explain to me why we should need it?
![image](https://user-images.githubusercontent.com/78462401/163824790-a77c5193-5f10-4d05-84ef-5103fc0b700c.png)

Reply:
I will approve your project because I'm not 100% sure about the project requirements. 
`-` signg for properties means they are read-only, So `id` and `items` should be read-only, otherwise, you can not create reports or preserve data.